weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -172,6 +167,11 @@ class SelectableCHCallSchema(BaseModel):
 
     deleted_at: typing.Optional[datetime.datetime] = None
 
+    attributes_dump: str
```

Comment:
we parse the returned sql based on where its returned, since we are modifying summary_dump, i moved it to the end

Reply:
This indicates that something is not quite correct. We should always be selecting an explicit set of columns, then directly reading those columns out.